.circleci/config.yml

```diff
@@ -659,6 +671,19 @@ jobs:
             docker push ${DOCKERHUB_REPO:?}:${CIRCLE_BRANCH:?}
       - *fail_notification
 
+  docker_push_latest:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    steps:
+      - checkout
+      - *setup_secrets
+      - *setup_remote_docker
+      - run:
+          name: Build and push Docker image to DockerHub
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            docker push ${DOCKERHUB_REPO:?}:latest
```

Comment:
Isn't it better to have explicit `docker tag ${DOCKERHUB_REPO:?}:${CIRCLE_TAG:?} ${DOCKERHUB_REPO:?}:latest` in this job instead of relying on `docker_push_tag` ?